students/csdotson/lesson04/mailroom_2.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+# Lesson 4 - Mailroom... part 2
+
+### Function Definitions ###
+def prompt(prompt, menu):
+    # Create menu based on arguments
+    while True:
+        response = input(prompt)
+        if menu[response]() == "quit":
+            break
+
+
+def send_thank_you():
+    # Create sub_prompt for creating thank you's and adding new donors/donations
+    prompt(thank_you_prompt, thank_you_menu)
+
+
+def add_donation():
+    # Append donation to existing donor, or add donor/donation to existing list
+    response = input("\nPlease enter a donor name: ")
+    if response not in donations.keys():
```

Comment:
`in` checks for key membership in dict, so this can be `if response not in donations:`